docs/targets/alibabaoss.md

```diff
@@ -0,0 +1,35 @@
+# Event Target for Alibaba OSS 
+
+This event target receives [CloudEvents][ce] over HTTP and sends them to [Alibaba OSS](https://www.alibabacloud.com/product/oss)
```

Comment:
How about following the same pattern as the `CloudEvents` reference by moving the Alibaba OSS URL to the bottom of the doc?

Reply:
done thank you